src/ec/packages/ec-component-text-input/text-input.html.twig

```diff
@@ -10,6 +10,7 @@
   - "name" (string) (default: '')
   - "type" (string) (default: 'text')
   - "width" (string) (default: '') 
+  - "extra_group_classes" (optional) (string) (default: '') Extra classes (space separated) for the text-input group
```

Comment:
i think we miss this in the md file for the text-input ;)

Reply:
done.